system/Test/CIDatabaseTestCase.php

```diff
@@ -135,6 +142,8 @@ public function __construct()
 	 */
 	public function setUp()
 	{
+		$this->loadDependencies();
```

Comment:
Okay, if don't mind the current code, I don't revert. It works.

Without changing the test database config, we can run all the rest of the tests:
~~~
$ ./phpunit --exclude-group DatabaseLive 
PHPUnit 5.3.4 by Sebastian Bergmann and contributors.

...............................................................  63 / 413 ( 15%)
............................................................... 126 / 413 ( 30%)
............................................................... 189 / 413 ( 45%)
............................................................... 252 / 413 ( 61%)
............................................................... 315 / 413 ( 76%)
............................................................... 378 / 413 ( 91%)
...................................                             413 / 413 (100%)

Time: 4.99 seconds, Memory: 6.00MB

OK (413 tests, 731 assertions)
~~~

If we run all the tests, only database live tests cause errors:
~~~
$ ./phpunit 
PHPUnit 5.3.4 by Sebastian Bergmann and contributors.

...............................................................  63 / 519 ( 12%)
............................................................... 126 / 519 ( 24%)
............................EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE 189 / 519 ( 36%)
EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE 252 / 519 ( 48%)
EEEEEEEE....................................................... 315 / 519 ( 60%)
............................................................... 378 / 519 ( 72%)
............................................................... 441 / 519 ( 84%)
............................................................... 504 / 519 ( 97%)
...............                                                 519 / 519 (100%)

Time: 5.16 seconds, Memory: 8.00MB

There were 106 errors:

1) CodeIgniter\Database\Live\CIDbTestCaseTest::testHasInDatabase
InvalidArgumentException: You have not selected a database type to connect to.

...

FAILURES!
Tests: 519, Assertions: 731, Errors: 106.
~~~